lib/searchkick/results.rb

```diff
@@ -137,46 +137,56 @@ def total_count
     alias_method :total_entries, :total_count
 
     def current_page
+      return if scroll_id
```

Comment:
What's the motivation for having pagination methods return `nil`? I think we can keep track of the page even with the scroll method.